lib/connection_string.js

```diff
@@ -500,6 +493,46 @@ function checkTLSOptions(queryString) {
   }
 }
 
+/**
+ * Checks options object if both options are present (any value) will throw an error.
+ *
+ * @param {Object} options Options Object
+ * @param {string} optionKeyA A options key
+ * @param {string} optionKeyB B options key
+ * @throws {MongoParseError}
+ */
+function assertRepelOptions(options, optionKeyA, optionKeyB) {
+  if (options.hasOwnProperty(optionKeyA) && options.hasOwnProperty(optionKeyB)) {
```

Comment:
(Just my preference, so feel free to resolve and ignore) `Reflect.has()` is available to us now same exact functionality just short and sweet 🍰

Reply:
🤩very nice, but using new things is scary.

Reply:
relevant: https://stackoverflow.com/a/58728360

Reply:
Not implementing, is that OK